dynamic-landing-page/js/main.js

```diff
@@ -0,0 +1,96 @@
+//DOM Elements
+const time = document.getElementById('time'),
+    greeting = document.getElementById('greeting'),
+    name = document.getElementById('name'),
+    focus = document.getElementById('focus');
+
+//Show Time
+function showTime(){
+    let today = new Date(),
+        hour = today.getHours(),
+        min = today.getMinutes(),
+        sec = today.getSeconds();
+    
+    hour = hour % 12 || 12;
+    const amPm = hour >=12 ? 'PM' : 'AM';
+    time.innerHTML = `${hour}<span>:</span>${addZero(min)}<span>:</span>${addZero(sec)} ${amPm}`;
+    setTimeout(showTime, 1000);
+};
+
+//Add Zero
+function addZero(n) {
+    return (n < 10 ? '0' : '') + n;
+}
+
+//Set Background and Greeting
+function setBgGreet() {
+    let today = new Date(),
+        hour = today.getHours();
+  
+    if (hour < 12) {
+      document.body.style.backgroundImage = "url('img/morning.jpg')";
+      greeting.textContent = 'Good Morning,';
+    } else if (hour < 18) {
+      document.body.style.backgroundImage = "url('img/afternoon.jpg')";
+      greeting.textContent = 'Good Afternoon,';
+    } else {
+      document.body.style.backgroundImage = "url('img/night.jpg')";
+      greeting.textContent = 'Good Evening,';
+      document.body.style.color = 'white';
+    }
+  }
+
+// Get Name
+function getName() {
+    if (localStorage.getItem('name') === null) {
+        name.textContent = '[Enter Name]';
+    } else {
+        name.textContent = localStorage.getItem('name');
+    }
+}
+
+//Set Name
+function setName(e) {
+    if(e.type = 'keypress') {
+        if(e.which == 13 || e.keyCode == 13) {
+            localStorage.setItem('name', e.target.innerText);
+            name.blur();
+        }
+    } else {
+        localStorage.setItem('name', e.target.innerText);
```

Comment:
Hi @andreiha the same you do in line 56. It's a point to optimize this function.